lms/templates/emails_change_successful.html

```diff
@@ -1,18 +0,0 @@
-<%inherit file="main.html" />
```

Comment:
@rlucioni Yes, that's the reason. I have searched for the file and haven't found any usage of the file.